charts/rancher-k3s-upgrader/0.3.0/templates/configmap.yaml

```diff
@@ -8,7 +8,7 @@ data:
   SYSTEM_UPGRADE_CONTROLLER_THREADS: {{ .Values.systemUpgradeControllerThreads | default "2" | quote }}
   SYSTEM_UPGRADE_JOB_ACTIVE_DEADLINE_SECONDS: {{ .Values.systemUpgradeJobActiveDeadlineSeconds | default "900" | quote }}
   SYSTEM_UPGRADE_JOB_BACKOFF_LIMIT: {{ .Values.systemUpgradeJobBackoffLimit | default "99" | quote }}
-  SYSTEM_UPGRADE_JOB_IMAGE_PULL_POLICY: {{ .Values.systemUpgradeJobImagePullPolicy | default "Always" | quote }}
+  SYSTEM_UPGRADE_JOB_IMAGE_PULL_POLICY: {{ .Values.systemUpgradeJobImagePullPolicy | default "IfNotPresent" | quote }}
```

Comment:
Is there an implication to updating `IfNotPresent` here rather than specifying `systemUpgradeJobImagePullPolicy: IfNotPresent` in the values.yaml?

Seems like this changes a default that is expected in the SUC chart; could this break users in certain environments?

Reply:
ah seems like Nick beat me to this in https://github.com/rancher/system-charts/pull/520/files#r691642993 and Dan responded. I will resolve this comment and approve!